apisix/admin/resource.lua

```diff
@@ -49,7 +49,41 @@ local function split_typ_and_id(id, sub_path)
 end
 
 
-function _M:check_conf(id, conf, need_id, typ)
+local function check_create_update_time(conf)
+    local not_allow_create_time = "forbidden create_time in request body"
+    local not_allow_update_time = "forbidden update_time in request body"
+
+    if conf.create_time then
+        return not_allow_create_time
+    end
+
+    if conf.update_time then
+        return not_allow_update_time
+    end
+
+    if conf.upstream then
+        if conf.upstream.create_time then
+            return not_allow_create_time
+        end
+        if conf.upstream.update_time then
+            return not_allow_update_time
+        end
+    end
+
+    if conf.plugins then
+        if conf.plugins.create_time then
+            return not_allow_create_time
+        end
+        if conf.plugins.update_time then
+            return not_allow_update_time
```

Comment:
Is it possible to use recursion to check all properties?

Reply:
I referred to other definitions containing subresources, such as ssl, and I found the benefits of this implementation

1. Simple to implement, easy to maintain, no recursion required.
2. Currently, only these two sub-resources are included in other resources.